### PR TITLE
Update copyright headers for dl4j modules

### PR DESCRIPTION
**Pull request by sshepel**
## What changes were proposed in this pull request?
Update copyright headers for:
- libnd4j
- jumpy
- dl4j
- arbiter

## How was this patch tested?
Tested manually on local VM.

## Quick checklist

The following checklist helps ensure your PR is complete:

- [ ] Reviewed the [Contributing Guidelines](https://github.com/deeplearning4j/deeplearning4j/blob/master/CONTRIBUTING.md) and followed the steps within.
- [ ] Created tests for any significant new code additions.
- [ ] Relevant tests for your changes are passing.
- [ ] Ran mvn formatter:format (see [formatter instructions](http://code.revelc.net/formatter-maven-plugin/examples.html#Setting_Source_Files) for targeting your specific files).

### CODE REVIEW

File: nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/graph/ByteOrder.java [hunk 1]
```diff
@@ -1,3 +1,19 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ******************************************************************************/
+
 // automatically generated by the FlatBuffers compiler, do not modify
```
Comment (saudet) [inline nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/graph/ByteOrder.java]: These files are generated by FlatBuffers, so it doesn't make sense to add them manually here. If we can't figure out how to make the generator also generate those, let's revert this.

File: nd4j/nd4j-common/src/main/java/org/nd4j/util/FingerPrintKeyer.java [hunk 2]
```diff
@@ -1,35 +1,18 @@
-/*
-
-Copyright 2010, Google Inc.
```
Comment (saudet) [inline nd4j/nd4j-common/src/main/java/org/nd4j/util/FingerPrintKeyer.java]: This file actually is under Google's copyright, we can't just replace it with our own!

**Comment by saudet** [comment]
Thanks @sshepel!

@maxpumperla @farizrahman4u The Python files in this PR have not been copied from somewhere and modified by us? In other words, can you confirm that they were created from scratch?

File: nd4j/nd4j-common/src/main/java/org/nd4j/util/FingerPrintKeyer.java (hunk 2 above)
Comment (sshepel) [reply nd4j/nd4j-common/src/main/java/org/nd4j/util/FingerPrintKeyer.java]: Oh, missed that! Thanks.

File: nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/graph/ByteOrder.java (hunk 1 above)
Comment (sshepel) [reply nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/graph/ByteOrder.java]: Hm, got it.
@raver119 do, you know if we can update the generator?

File: nd4j/nd4j-common/src/main/java/org/nd4j/util/FingerPrintKeyer.java (hunk 2 above)
Comment (sshepel) [reply nd4j/nd4j-common/src/main/java/org/nd4j/util/FingerPrintKeyer.java]: Fixed.

**Pull request merged**